### DISCUSSION TITLE
How to search a bib entry based on article pages number?

### DISCUSSION THREAD

**Discussion by nujabse** [emacs-citar/citar#768] [Q&A] [answered]
Sometimes, I would search a bib entry by its page number, however, this does not seem to work in default settings for citar. Searching using bib key, title and year works, though. Can citar by configured to use pages number for searches?

**Comment by bdarcus** [comment, marked as answer]
You can only search on fields you include in your `main` and `suffix` templates. So you'd want to add it, probably to the latter.

The only exception is we include some hidden data that allows you to search for but not see, mostly so you can filter based on the indicators.

**Reply by nujabse** [reply]
Thanks for your reply. I have changed the `suffix` templates, now I can search by pages.
